src/components/elections-stats/elections-stats.ce.vue

```diff
@@ -135,7 +135,7 @@ export default {
               accommodations = accommodations.slice(0, self.chartSize);
             }
             self.graphs.push({
-              title: accommodations.length + " accommodations",
+              title: accommodations.length + " colleges",
```

Comment:
Leaving the prop as accommodations as it matches what is received in the API and colleges refers to faculties in SUMS